frontend/app/views/fragments/bundle/bundleButton.scala.html

```diff
@@ -0,0 +1,25 @@
+
+@(buttonLabel: String, target: String, dataLink: String, chevron: Boolean = false)
+
+<span class="bundle-button">
+    <a data-link-name="@dataLink" class="link-button" href="@target">
+        <div class="bundle-button__content">
+            <p>@buttonLabel</p>
+            <span class="bundle-button__content__arrow">
```

Comment:
If we make this `display: inline` in the CSS, can we use `div` instead of `span`?